docs/snippets/vue/table-story-fully-customize-controls.vue3.js.mdx

```diff
@@ -0,0 +1,28 @@
+```js
+// Table.stories.js
+
+const TableStory = (args) => ({
+  components: { Table },
+  setup() {
+    return { args };
+  },
+  template: `
+    <tr v-for="(row, idx1) in data">
```

Comment:
I think `data` needs to be accessed from args?

Reply:
With the latest commits the example is fixed. The `Table` component required to included